src/schema/v2/me/index.ts

```diff
@@ -155,6 +155,46 @@ export const meType = new GraphQLObjectType<any, ResolverContext>({
     },
     conversation: Conversation,
     conversationsConnection: Conversations,
+    counts: {
+      type: new GraphQLObjectType<any, ResolverContext>({
+        name: "MeCounts",
+        fields: {
+          followedArtists: {
+            type: new GraphQLNonNull(GraphQLInt),
```

Comment:
#minor but having a non-null, and then a loader w/o try/catch (yada yada - we've discussed before!), can wind up error'ing 'loudly', and the entire `counts: null` would be null.

since we don't care about surfacing these errors, maybe it should be `try/catch` with the `catch` returning 0? same below

Reply:
Ah yep, try/catch indeed.